### PR TITLE
feat: merge landing page and UI + pre-mining rewards UI

### PR DESCRIPTION
**Pull request by keeplearning-near**

### CODE REVIEW

File: src/ui-config/markets/index.ts [hunk 1]
```diff
@@ -29,11 +30,29 @@ export const marketsData: { [key in keyof typeof CustomMarket]: MarketDataType }
...
     addresses: {
-      LENDING_POOL_ADDRESS_PROVIDER: '0x56b2D779eD5BEb5AE05907653c4C3a4FdD0eE928'.toLowerCase(),
-      LENDING_POOL: '0x03b6314535Ce2d9c2dc743A8D900bbbE6D48215C',
-      WETH_GATEWAY: '0xe7E3F739AE4cB348Eb178576BFFBD19584878877',
+      LENDING_POOL_ADDRESS_PROVIDER: '0x8fcc9f65BF153371f4B06542446eE51aCcDb635C'.toLowerCase(),
+      LENDING_POOL: '0x8ca1db9e43305636e9b4bf8ca1ff8f97579009ad',
+      WETH_GATEWAY: '0x33318ba00a220cab6da06f625c0d80ad5a2f37a9',
       // SWAP_COLLATERAL_ADAPTER: '0x2EcF2a2e74B19Aab2a62312167aFF4B78E93B6C5',
     },
+    cornRewards: {
```
Comment (linguists) [inline src/ui-config/markets/index.ts:38]: Shouldn't these be configured via contract?

File: src/ui-config/networks.ts [hunk 2]
```diff
@@ -194,21 +222,34 @@ export const networkConfigs: Record<string, BaseNetworkConfig> = {
...
+    rewardTokens: [
+      {
+        rewardTokenSymbol: 'WNEAR',
+        rewardTokenAddress: '0x1a55e008e86ac4c170499e8e8af5d60565e4e453',
+        rewardTokenDecimals: 24,
+      },
+      {
+        rewardTokenSymbol: 'CORN',
+        rewardTokenAddress: '0xa4904872b5b17d3101857c649a8e7e7a3ed5b1ac',
+        rewardTokenDecimals: 26,
```
Comment (linguists) [inline src/ui-config/networks.ts:250]: Do we need to keep decimals as `26`?

File: src/modules/dashboard/screens/Dashboard/index.tsx [hunk 3]
```diff
@@ -47,6 +49,8 @@ export default function Dashboard() {
   const { chainId } = useProtocolDataContext();
   const { user, reserves } = useDynamicPoolDataContext();
   const { reserveIncentives } = useIncentivesDataContext();
+  // TODO: reserveIncentive need to map
```
Comment (keeplearning-near) [inline src/modules/dashboard/screens/Dashboard/index.tsx:52]:
```suggestion
```

File: src/modules/borrow/screens/BorrowMain/index.tsx [hunk 4]
```diff
@@ -30,6 +30,8 @@ export default function BorrowMain() {
   const { marketRefPriceInUsd } = useStaticPoolDataContext();
   const { reserves, user } = useDynamicPoolDataContext();
   const { reserveIncentives } = useIncentivesDataContext();
+  // TODO: reserveIncentive need to map
```
Comment (keeplearning-near) [inline src/modules/borrow/screens/BorrowMain/index.tsx:33]:
```suggestion
```

**Review by linguists — APPROVED** [review]

**Pull request merged**